src/templates/default/reviews/_includes/proposal_table.html

```diff
@@ -40,7 +40,13 @@
 			{% if proposal.accepted == None %}
 				<td>{{ proposal.review_count }}</td>
 			{% endif %}
-            <td class="proposal-title"><div>{{ proposal.title }}</div></td>
+            <td class="proposal-title">
+							{% if proposal.review_count > reviews_enough %}
+								<div style="color: darkgray">{{ proposal.title }}</div>
+							{% else %}
+								<div>{{ proposal.title }}</div>
+							{% endif %}
```

Comment:
I think you can put the {% if %}...{% else %}...{% endif %} clause inside the div, something like
```html
<div{% if ... %} class="grayed-out"{% endif %}>...
```
and maybe add a CSS class for this grayed-out color? Take a look at [src/static/css/pages/_proposals.scss](src/static/css/pages/_proposals.scss)

Reply:
Ok，I will modify